src/vellum/workflows/ports/port.py

```diff
@@ -61,6 +61,25 @@ def edges(self) -> Iterator[Edge]:
         return iter(self._edges)
 
     def __rshift__(self, other: GraphTarget) -> Graph:
+        # Check for trigger target first (both class and instance)
+        from vellum.workflows.triggers.base import BaseTrigger
+
+        # Check if other is a trigger class
+        if isinstance(other, type) and issubclass(other, BaseTrigger):
+            raise TypeError(
+                f"Cannot create edge targeting trigger {other.__name__}. "
+                f"Triggers must be at the start of a graph path, not as targets. "
+                f"Did you mean: {other.__name__} >> {self.node_class.__name__}?"
+            )
+
+        # Check if other is a trigger instance
+        if isinstance(other, BaseTrigger):
+            raise TypeError(
+                f"Cannot create edge targeting trigger {other.__class__.__name__}. "
+                f"Triggers must be at the start of a graph path, not as targets. "
+                f"Did you mean: {other.__class__.__name__}() >> {self.node_class.__name__}?"
```

Comment:
[blocking] can we keep working with just BaseTrigger classes instead of instances? just to keep consistent with how our nodes work